indexer/covidResearchPaperConnector/src/main/resources/covid_metadata.csv

```diff
@@ -0,0 +1,11 @@
+cord_uid,title,uri,url,authors
```

Comment:
maybe you can add two files, covid_metadata_small.csv and covid_metadata_large.csv

Reply:
done